### PR TITLE
updated links to diagrams in the README

### PR DESCRIPTION
**Pull request by laUmar123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ For a visual representation of the project architecture and details on the desig
 
 ### Architecture Diagram
 
-![Architecture Diagram](https://github.com/fm1psy/c11-poseidon-earthquake-monitoring/blob/main/diagrams/earthquake_architecture.png)
+![Architecture Diagram](https://github.com/fm1psy/c11-poseidon-earthquake-monitoring/blob/main/diagrams/architecture-diagram.png)
```
Comment (fm1psy) [inline README.md:21]: I noticed you changed the filename at the end of the image url. Is there a reason you replaced the _ with a dash?

**Review by joe1606 — APPROVED** [review]
👍

**Review by fm1psy — COMMENTED** [review]
I left a question here as I just wanted some clarification on the changes you made

File: README.md (hunk 1 above)
Comment (laUmar123) [reply README.md:21]: No real reason, I'll change it to '_' for consistency.

**Review by fm1psy — APPROVED** [review]
Looks good, thanks

**Pull request merged**